docs/source/technical_design/configoois.rst

```diff
@@ -0,0 +1,42 @@
+======================
+What are Config OOI's?
+======================
+
+Config OOI's are, in principle, the same as regular objects, but serve to apply settings to the object that they to.
```

Comment:
Hmm not necessarily settings. Also preferences about underlying OOIs. They sort of serve as extra input for specific bits that run on underlying objects.